packages/formik/src/FieldArray.tsx

```diff
@@ -155,26 +153,6 @@ class FieldArrayInner<Values = {}> extends React.Component<
     this.pop = this.pop.bind(this) as any;
   }
 
-  shouldComponentUpdate(props: any) {
```

Comment:
Did you intend to delete this? (although this is actually breaking my code when a component returned by the `render` function depends on local state.)

Reply:
I intended to revert the change yes.